.github/workflows/python-tests.yml

```
@@ -8,34 +6,40 @@ defaults:
     working-directory: ./python
 
 jobs:
-  python_functional_tests:
-    name: Python Package Functional Tests
+  python_tests:
+    name: Python ${{ matrix.py_version }} Tests
     runs-on: ubuntu-latest
 
-    outputs:
-      wheel_name: ${{ steps.wheel_artifact.outputs.wheel_name }}
+    strategy:
+      matrix:
+        py_version: ["3.9.0", "3.9.13", "3.10.4"]
```

Per our previous discussion, I think we want this to be 3.10.5.

```suggestion
        py_version: ["3.9.0", "3.9.13", "3.10.5"]
```